link.html

```diff
@@ -130,7 +130,7 @@ <h3 id="searchlabel">Quick search</h3>
             <!-- Source interaction buttons -->
 
 <div class="dropdown-buttons-trigger">
-    <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn"
+    <button id="sourc-repo-dropdown-buttons-trigger" class="btn btn-secondary topbarbtn"
```

Comment:
Typo on source.

Not sure about the purpose of this change, because there was a duplicate id? Is the id needed at all on these buttons?

Reply:
they did not look needed but I didn't think I should just remove it.

Reply:
fixed the misspelling